src/certs/x509.ts

```diff
@@ -53,7 +53,7 @@ export class X509Certificate extends CryptoCertificate implements core.CryptoX50
     const array = new Uint8Array(data);
     this.parse(array.buffer as ArrayBuffer);
 
-    const { token, label, sensitive, ...keyAlg } = algorithm; // remove custom attrs for key
+    const { token, label, sensitive, ...keyAlg } = algorithm as any; // remove custom attrs for key
```

Comment:
We should not use `any`. Looks like you need `id` filed in algorithm. It would be better to add this field into the algorithm interface